src/overlays/actors/ovl_En_Nb/z_en_nb.h

```diff
@@ -2,18 +2,56 @@
 #define Z_EN_NB_H
 
 #include "global.h"
+#include "objects/object_nb/object_nb.h"
 
 struct EnNb;
 
 typedef void (*EnNbActionFunc)(struct EnNb*, PlayState*);
+typedef s32 (*EnNbUnkFunc)(Actor*, PlayState*);
+
+typedef enum EnNbAnimation {
+    /* -1 */ EN_NB_ANIM_INVALID = -1,
+    /*  0 */ EN_NB_ANIM_0,
+    /*  1 */ EN_NB_ANIM_1,
+    /*  2 */ EN_NB_ANIM_TALK_ONCE,
+    /*  3 */ EN_NB_ANIM_TALK_LOOP,
+    /*  4 */ EN_NB_ANIM_ANGRY,
+    /*  5 */ EN_NB_ANIM_RELIEVED,
+} EnNbAnimation;
 
 typedef struct EnNb {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x44];
+    /* 0x144 */ SkelAnime skelAnime;
     /* 0x188 */ EnNbActionFunc actionFunc;
-    /* 0x18C */ char unk_18C[0x110];
+    /* 0x18C */ EnNbActionFunc unk_18C;
+    /* 0x190 */ ColliderCylinder collider;
+    /* 0x1DC */ u8 scheduleResult;
+    /* 0x1E0 */ UNK_PTR unk_1E0;
```

Comment:
This should be a `u8*`; the prototype is wrong atm. You can probably name the other stuff the msgEvent function uses from the bjt uses if only to be consistent.